recipes/addons.rb

```diff
@@ -67,10 +67,9 @@
   node.override['kube-hops']['docker_img_reg_url'] = registry_host + ":#{node['hops']['docker']['registry']['port']}"
 end
 
+include_recipe "kube-hops::hops-system"
+include_recipe "kube-hops::filebeat"
+
 if node['kube-hops']['kfserving']['enabled'].casecmp?("true")
   include_recipe "kube-hops::kfserving"
```

Comment:
Do we need an else statement here?

else
 node.override['kube-hops']['filebeat']['enabled'] = "false"
end

Reply:
Filebeat is independent from KFServing. It is also used to harvest logs from the standard Kubernetes deployments (our default serving tool). Thus, if Kubernetes is installed, filebeat needs to be installed as well. 

I removed the attribute _kube-hops/filebeat/enabled_.

Reply:
Another option is to make the installation of filebeat optional (via _kube-hops/filebeat/enabled_), but then if it's **false** server logs will not be collected when models are served on Kubernetes (either using normal deployments or kfserving)